src/routes/api/[...endpoint]/proxy.ts

```diff
@@ -0,0 +1,71 @@
+import { PUBLIC_API_URL } from '$env/static/public';
+import type { Cookies } from '@sveltejs/kit';
+import { JWT_REFRESH_KEY, JWT_ACCESS_KEY, appendCookieHeader } from '$lib/auth';
+
+async function _fetchAdapter(
```

Comment:
This function is used a lot, so it's nice to have it but I don't entirely understand its goal. Is it just to prepend the API URL and add the json header? If so, maybe we should have the name reference API in some way?

Reply:
Changed the name accordingly